packages/frontend/src/components/nft/NFTDetail.js

```diff
@@ -0,0 +1,225 @@
+import React, { useEffect, useState } from 'react';
+import { Translate } from 'react-localize-redux';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { useFungibleTokensIncludingNEAR } from '../../hooks/fungibleTokensIncludingNEAR';
+import UserIconGrey from '../../images/UserIconGrey';
+import { selectAccountId } from '../../redux/slices/account';
+import NonFungibleTokens from '../../services/NonFungibleTokens';
+import FormButton from '../common/FormButton';
+import Container from '../common/styled/Container.css';
+import ArrowIcon from '../svg/ArrowIcon';
+import NFTTransferModal from './NFTTransferModal';
+
+const StyledContainer = styled(Container)`
+    .container {
+        margin-left: auto;
+        margin-right: auto;
```

Comment:
I believe `../common/styled/Container.css` should be enough to center the container? You can add additional variations to `Container.css` if needed so that we can re-use it at a later point if needed. can probably remove the `.container` `div` afterwards